packages/sdk-codegen/src/go.gen.spec.ts

```diff
@@ -34,6 +34,41 @@ const gen = new GoGen(apiTestModel)
 const indent = ''
 
 describe('Go generator', () => {
+  describe('bookends', () => {
+    it('has a v3 models prologue', () => {
+      const origPkgName = gen.packageName
+      gen.packageName = gen.apiVersion = 'v3'
+      const prologue = gen.modelsPrologue('')
+      gen.packageName = gen.apiVersion = origPkgName
```

Comment:
didn't really dig too deep into how `apiVersion` and `packageName` are set but this works